pipeline_api/src/test/java/com/splicemachine/pipeline/PipelineTest.java

```diff
@@ -50,6 +51,7 @@
  * @author Scott Fines
  *         Date: 12/23/15
  */
+@Ignore // doesn't work (No PipelineTestDataEnv found)
```

Comment:
I think you can run it by specifying either cdh or mem in the classpath of the test run configuration (if you're using IntelliJ), please make sure that the test is not running in Jenkins before ignoring it otherwise this PR decreases test coverage.

Reply:
The test is not running nor working on current master. We should make them run again and then enable in jenkins/pom (DB-11686).

Reply:
ok, i removed this commit